htdocs/webservices/server_invoice.php

```diff
@@ -243,13 +243,37 @@
     // Entry values
     array('authentication'=>'tns:authentication','invoice'=>'tns:invoice'),
     // Exit values
-    array('result'=>'tns:result','id'=>'xsd:string','ref'=>'xsd:string'),
+    array('result'=>'tns:result','id'=>'xsd:string','ref'=>'xsd:string','ref_ext'=>'xsd:string'),
     $ns,
     $ns.'#createInvoice',
     $styledoc,
     $styleuse,
     'WS to create an invoice'
 );
+$server->register(
```

Comment:
Implement new method is no more possible into 3.7 branch. 
Please commit this and any new feature (not bugfix) into develop branch.
Changes made into 3.7 will be merged into develop branch.